packages/react-server-examples/bike-share/pages/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {ReactServerAgent, RootContainer, RootElement} from 'react-server';
+import NetworkList from '../components/network-list';
+import Header from '../components/header';
+import Footer from '../components/footer';
+
+export default class IndexPage {
+	handleRoute(next) {
+		this.data = ReactServerAgent.get('http://api.citybik.es/v2/networks');
+		return next();
+	}
+	getElements() {
+		return [
+			<RootContainer key={0}>
```

Comment:
I added the key because of [this eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md) that comes from [xo-react](https://github.com/sindresorhus/eslint-config-xo-react).  If I understand correctly, its to address [this  jsx gotcha](https://facebook.github.io/react/docs/jsx-gotchas.html#html-entities).  I believe it addresses the `Each child in an array should have a unique "key" prop.` message I see so often in our React Server implementation.